lua/entities/base_wire_entity.lua

```diff
@@ -384,7 +425,9 @@ function ENT:OnEntityCopyTableFinish(dupedata)
 	-- Called by Garry's duplicator, to modify the table that will be saved about an ent
 
 	-- Remove anything with non-string keys, or util.TableToJSON will crash the game
-	dupedata.OverlayData_UpdateTime = nil
+	dupedata.OverlayData = nil
+	dupedata.lastWireOverlayUpdate = nil
+	dupedata.playersRequestingOverlayNumeric = nil
```

Comment:
This has to be obsolete/unneeded by now, right? @Divran

Reply:
I dunno, I havent looked at how the new code works in enough detail to tell. In any case, deleting those from the dupe wouldn't do any harm, even if it's not necessary.

Reply:
removing uneeded data wont hurt, it may even help, nothing wrong with a smaller file.